habits/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import permissions
+
+
+class IsOwner(permissions.BasePermission):
+    def has_object_permission(self, request, view, obj):
+        if obj.user == request.user:
```

Comment:
А если нет - вернется None

Тут правильнее написать так:

```suggestion
        return obj.user == request.user:
```
Тогда всегда будет возвращаться True/False

Ниже тоже нужно поправить (пермишен не должен возвращать None)